src/x/context/context.go

```diff
@@ -313,10 +317,10 @@ func (c *ctx) spanIsSampled(sp opentracing.Span) bool {
 	return true
 }
 
-func (c *ctx) StartTraceSpan(name string) (Context, opentracing.Span, bool) {
+func (c *ctx) StartSampledTraceSpan(name string) (Context, opentracing.Span, bool) {
```

Comment:
nit: add comments to exported functions. write a blurb bout using StartSampledTraceSpan v the other one.

Reply:
oh nvm, saw you have them in the interface def

Reply:
I thought the convention was to just comment on the interface?